valve-monitor/web-app/src/styles/ValveMonitorTable.module.scss

```diff
@@ -51,3 +63,95 @@
     margin: auto;
   }
 }
+
+@media screen and (max-width: 910px) {
+  .tableContainer {
+    margin-bottom: 32px;
+
+    :global .ant-table-thead .ant-table-cell,
+    :global .ant-table-tbody td {
+      font-size: 14px;
+    }
+
+    :global .ant-table-tbody .editable-input-name,
+    :global .ant-table-tbody .editable-button-name {
+      max-width: 100px;
+      width: 100px;
+    }
+  }
+}
+
+@media screen and (max-width: 708px) {
+  .tableContainer {
+    border: none;
+
+    :global .ant-table-thead .ant-table-cell {
+      font-size: 12px;
+      display: none;
+    }
+
+    .rowTitle {
+      display: block;
+      width: 50%;
+      font-weight: 600;
+    }
+
+    :global .ant-table-tbody tr {
+      display: flex;
+      flex-direction: column;
+      margin-bottom: 24px;
+      width: 117%; // I don't know why this is necessary but it is
```

Comment:
> // I don't know why this is necessary but it is

😄

Reply:
otherwise the row wouldn't span the width of the card. So odd